content/en/docs/topics/registries.md

```diff
@@ -1,30 +1,63 @@
 ---
-title: "Registries"
+title: "Use OCI-based registries"
```

Comment:
Are we sure we want to mention OCI in the title? The reason I ask is because non-OCI registries are also in scope for this doc in general.

Reply:
Ah ok, I thought this was only oci-based registries. Is there any difference in setup or usage besides the URL prefix when referencing artifacts? I'm assuming charts stored in non-oci registries are a different format?

Reply:
I'm going to ask @jdolitsky to weigh in here, as he has the best understanding of this update - thanks.

Reply:
hello! This doc should be covering only OCI-based registries, so the change makes sense to me. I think either title would be reasonable.